.eslintignore

```diff
@@ -0,0 +1,2 @@
+tests.js
+scoring.js
```

Comment:
Hmmm.. interested to know why you had to add `scoring.js` here as well. That file should have been in a good state and adding it here would just mean you don't get the benefits of ESLint. Was it a problem with newlines?